cirq/protocols/json_test_data/Circuit.json

```diff
@@ -170,7 +171,8 @@
     ],
     "device": {
       "cirq_type": "_UnconstrainedDevice"
-    }
+    },
+    "name": null
```

Comment:
consider omitting this field in the json output when it's not set for fully-backwards-compatible output and generally nicer json

Reply:
Done.